### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#466] [Esami & Esoneri M-Z]
### Traccia

<img width="568" alt="image" src="https://github.com/user-attachments/assets/cd3aea44-043e-4697-8f44-ab701a943a16">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/61b9d965-2bf9-49e4-be43-520b920ee303)
